reference/compiling_for_android.rst

```diff
@@ -16,93 +16,81 @@ Also, you still need to do all the steps mentioned in the
 :ref:`doc_exporting_for_android` tutorial before attempting your custom
 export template.
 
-Requirements
-------------
+A. Requirements
```

Comment:
It's up to you and the guidelines.

From my experience in guide writing, my style usually easier to understand. It's ok to not believe it. Just something like Code Folding, easier to grasp the whole concept/idea of a script.

But, again... If you can't accept it, I will follow previous style / yours.

:-)